[2024-05-17T05:40:30.556+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: temp_humi_dag.delete_data manual__2024-05-17T05:40:29.558000+00:00 [queued]>
[2024-05-17T05:40:30.564+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: temp_humi_dag.delete_data manual__2024-05-17T05:40:29.558000+00:00 [queued]>
[2024-05-17T05:40:30.565+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 4
[2024-05-17T05:40:30.575+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): delete_data> on 2024-05-17 05:40:29.558000+00:00
[2024-05-17T05:40:30.581+0000] {standard_task_runner.py:60} INFO - Started process 4565 to run task
[2024-05-17T05:40:30.584+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'temp_humi_dag', 'delete_data', 'manual__2024-05-17T05:40:29.558000+00:00', '--job-id', '167', '--raw', '--subdir', 'DAGS_FOLDER/ICTTemp_Humi_DAG.py', '--cfg-path', '/tmp/tmpu1d76532']
[2024-05-17T05:40:30.587+0000] {standard_task_runner.py:88} INFO - Job 167: Subtask delete_data
[2024-05-17T05:40:30.636+0000] {task_command.py:423} INFO - Running <TaskInstance: temp_humi_dag.delete_data manual__2024-05-17T05:40:29.558000+00:00 [running]> on host e3e2dc6bc2be
[2024-05-17T05:40:30.696+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Group4' AIRFLOW_CTX_DAG_ID='temp_humi_dag' AIRFLOW_CTX_TASK_ID='delete_data' AIRFLOW_CTX_EXECUTION_DATE='2024-05-17T05:40:29.558000+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-05-17T05:40:29.558000+00:00'
[2024-05-17T05:40:30.702+0000] {logging_mixin.py:188} INFO - JSON data deleted successfully.
[2024-05-17T05:40:30.710+0000] {base.py:83} INFO - Using connection ID 'postgres_conn' for task execution.
[2024-05-17T05:40:30.711+0000] {logging_mixin.py:188} INFO - Error: could not translate host name "172.20.10.2 host.docker.internal" to address: Name or service not known
[2024-05-17T05:40:30.712+0000] {python.py:201} INFO - Done. Returned value was: None
[2024-05-17T05:40:30.720+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=temp_humi_dag, task_id=delete_data, execution_date=20240517T054029, start_date=20240517T054030, end_date=20240517T054030
[2024-05-17T05:40:30.759+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-05-17T05:40:30.777+0000] {taskinstance.py:3280} INFO - 1 downstream tasks scheduled from follow-on schedule check
